package ru.swiftail.dataobjects.api.exceptions;

public class TargetIOException extends DataObjectsException {
    public TargetIOException(String message, Throwable cause) {
        super(message, cause);
    }

    public TargetIOException(Throwable cause) {
        super(cause);
    }

    public TargetIOException(String message) {
        super(message);
    }
}
